docs/appendices/release-notes/5.5.3.rst

```diff
@@ -49,7 +49,7 @@ See the :ref:`version_5.5.0` release notes for a full list of changes in the
 Fixes
 =====
 
-- Forbid :ref:`renaming <sql-alter-table-rename-to>` of views and tables if the
+- Forbade :ref:`renaming <sql-alter-table-rename-to>` of views and tables if the
```

Comment:
```suggestion
- Rejected :ref:`renaming <sql-alter-table-rename-to>` of views and tables if the
```

Reply:
couldn't come up with the correct verb, thx!!!